timeHelper.go

```diff
@@ -6,19 +6,13 @@ type timeWrapper interface {
 	Now(*time.Location) time.Time
 	Unix(int64, int64) time.Time
 	Sleep(time.Duration)
-	Date(int, time.Month, int, int, int, int, int, *time.Location) time.Time
 	NewTicker(time.Duration) *time.Ticker
 }
 
-func newTimeWrapper() timeWrapper {
-	return &trueTime{}
-}
-
 type trueTime struct{}
 
 func (t *trueTime) Now(location *time.Location) time.Time {
-	n := time.Now().In(location)
-	return t.Date(n.Year(), n.Month(), n.Day(), n.Hour(), n.Minute(), n.Second(), 0, location)
+	return time.Now().In(location)
```

Comment:
Wasn't there a valid reason for how it was earlier? Have a look at his commit: 46d7545b76cfab41dc5616b015e08fa91e43df01

Reply:
I ran a per second job and for 5 minutes and it never drifted - IIRC this was related to that bug but is not itself an issue.